wordpress-mobile/react-native-aztec
# Disable history in AztecAndroid

This PR disables the History feature in the underlying Aztec-Android component since we're not going to use it in ReactNative/GB-mobile.

History will be handled JS side, and removing it from the underlying Aztec editor should improve performances - less calls to the internal `toHTML` method, and the history TextWatcher is also by-passed.

**To test**
Put a breakpoint here https://github.com/wordpress-mobile/react-native-aztec/blob/remove-history-aztec-android/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java#L52
and check that both `history` == 0 and `historyEnable` == false.

Note that i've also removed a layout file not used anywhere.
